taxcalc/params.json

```diff
@@ -920,9 +920,9 @@
     },
     "_NIIT_trt":{
         "long_name": "Net Investment Income Tax rate (UIMC)",
-	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed on net investment income of individuals with MAGI higher than certain thresholds.",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Also called Unearned Income Medicare Contribution Tax. You will be taxed at this rate if your MAGI is higher than NIIT threshold.",
```

Comment:
> I would suggest:  "Also called the Unearned Income Medicare Contribution
Tax. The lessor of net investment income and the amount by which MAGI
exceeds a threshold is taxed at this rate."

Very nice, although I'd drop the 'tax' from Unearned Income Medicare Contribution (I've never heard tax added at the end of UIMC). 

>Should we strive to eliminate the "you" and "your" from these
descriptions? Shouldn't it be "taxpayer"?

Also a good point. We should use "taxpayer" throughout.